core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/customize-interaction-modal.template.html

```diff
@@ -17,7 +17,8 @@ <h3 ng-if="StateInteractionIdService.displayed && (customizationModalReopened ||
         <br>
         <div ng-repeat="interactionId in category.interaction_ids"
              ng-click="onChangeInteractionId(interactionId)"
-             class="oppia-interaction-tile protractor-test-interaction-tile-<[interactionId]>">
+             class="oppia-interaction-tile protractor-test-interaction-tile-<[interactionId]>"
+             ng-if="interactionId !== 'RatioAExpressionInput'">
```

Comment:
Incorrect spelling.

Reply:
done

Reply:
Still not done? It's not `RatioAExpressionInput`, right?
Have you verified locally that this is not visible in the customize interaction modal?